TypeCobol.Test/Parser/Programs/Cobol85/WhenTestNoEndEvaluate.rdz.cbl

```diff
@@ -0,0 +1,34 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. WhenTest.
+       data division.
+       working-storage section.
+       01 var1 pic X(5) value 'hi'.
+       01 var2 pic X(5) value 'ho'.
+       01 var3 pic X(5) value 'ho'.
+       01 bool1 pic 9.
+       01 bool2 pic 9.
+       PROCEDURE DIVISION.
+           evaluate true
+                when
+                when bool1 display "evaluate"
+                when bool1
+                     when bool2 display "evaluate"
+                when
+                when other continue
+           evaluate true
+                when bool1 display "evaluate"
+                when bool1
+                when other continue
+           search var1
+                when
+                when var2
+                    when var3 display "search"
+                when
+           end-search
+           evaluate true
+                when bool1 display "evaluate"
+      *KO missing statement in "when other" clause
```

Comment:
Without `end-evaluate`, this `when other` is not empty anymore.
Can you change the comment ?

Reply:
done in 
75d47f4